mscore/musescore.cpp

```diff
@@ -5879,6 +5879,12 @@ void MuseScore::cmd(QAction* a)
                tr("Command %1 not valid in current state").arg(cmdn));
             return;
             }
+      if (sc->assignedWidget() == MsWidget::SCORE_TAB
+         && QApplication::focusWidget()
```

Comment:
So this additional condition fixes the Travis mtest hang? Black Magic ti me...

Reply:
I figured there might not be a widget that has focus during the tests since there's no GUI. It was just a guess though.

Reply:
An excellent one it seems, an 'educated guess' I'd say ;-)